.github/workflows/label-actions.yml

```diff
@@ -5,11 +5,14 @@ on:
     types: [labeled]
   discussion:
     types: [labeled]
+  pull_request:
```

Comment:
This action was last updated on Oct 1, 2021, that's more than a year ago! Do we want to keep using this action?

https://github.com/dessant/label-actions/releases

Reply:
sure, as long as it works as expected